modules/serverless-cluster-sizing-requirements.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+
+[id="cluster-size-requirements_context"]
+= Cluster size requirements
+
+Determine how many nodes and pods you require for your OpenShift Container Platform cluster. Cluster scalability correlates to the number of pods in a cluster environment. That number influences the other numbers in your setup.
+
+// Scalability, planning requirements, and limits
+include:scalability_and_performance/planning-your-environment-according-to-object-limits.adoc[leveloffset=+1]
+
+Your cluster must be sized appropriately to ensure the instructions work properly. To do so, you can use the MachineSet API to manually scale your cluster up to the desired size.
+
+An Openshift cluster with 16 CPUs and 40 GB memory is the minimum requirement for getting started with your first serverless application. This usually means you will have to scale up one of the default MachineSets by two additional machines.
+
+For more advanced use-cases, such as using Openshift Logging, Monitoring, Metering and Tracing, you will need to deploy more resources. Recommended requirements for such use-cases are 24 vCPUs and 96GB of memory.
+
+// Add or remove an instance of a machine in a MachineSet
```

Comment:
There isn't enough content here about what machinesets are IMHO, maybe add a link to some context info or add something about how these work regarding serverless?